### ISSUE TITLE
Version 2.0.1234 Object Reference Errors - Doesn't stop it from working though.

### ISSUE THREAD

**Issue by Nurvx** [tidusjar/Ombi#942]
#### Ombi Version:

V 2.0.1234

#### Update Branch:

Stable

#### Operating System:

Windows 7/Windows Server 2008 R2

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

System.NullReferenceException: Object reference not set to an instance of an object. at Ombi.Services.Jobs.PlexEpisodeCacher.CacheEpisodes(PlexSettings settings) in C:\projects\requestplex\Ombi.Services\Jobs\PlexEpisodeCacher.cs:line 141 at Ombi.Services.Jobs.PlexEpisodeCacher.Execute(IJobExecutionContext context) in C:\projects\requestplex\Ombi.Services\Jobs\PlexEpisodeCacher.cs:line 164	Ombi.Services.Jobs.PlexEpisodeCacher	Error	1/16/2017 10:54:41 AM
System.InvalidOperationException: There is an error in XML document (1, 2). ---> System.InvalidOperationException: was not expected. at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderPlexLibraries.Read4_MediaContainer() --- End of inner exception stack trace --- at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events) at System.Xml.Serialization.XmlSerializer.Deserialize(TextReader textReader) at Ombi.Api.ApiRequest.DeserializeXml[T](String input) in C:\projects\requestplex\Ombi.Api\ApiRequest.cs:line 134	Ombi.Api.ApiRequest	Error	1/16/2017 10:54:41 AM
Ninject.ActivationException: Error activating SonarrCacher No matching bindings are available, and the type is not self-bindable. Activation path: 1) Request for SonarrCacher Suggestions: 1) Ensure that you have defined a binding for SonarrCacher. 2) If the binding was defined in a module, ensure that the module has been loaded into the kernel. 3) Ensure you have not accidentally created more than one kernel. 4) If you are using constructor arguments, ensure that the parameter name matches the constructors parameter name. 5) If you are using automatic module loading, ensure the search path and filters are correct. at Ninject.KernelBase.Resolve(IRequest request) at Ninject.ResolutionExtensions.GetResolutionIterator(IResolutionRoot root, Type service, Func`2 constraint, IEnumerable`1 parameters, Boolean isOptional, Boolean isUnique) at Ombi.UI.Helpers.ServiceLocator.Resolve(Type type) in C:\projects\requestplex\Ombi.UI\Helpers\ServiceLocator.cs:line 54 at Ombi.UI.Jobs.CustomJobFactory.NewJob(TriggerFiredBundle bundle, IScheduler scheduler) in C:\projects\requestplex\Ombi.UI\Jobs\CustomJobFactory.cs:line 70	Ombi.UI.Jobs.CustomJobFactory	Error	1/16/2017 10:44:15 AM

```

#### Problem Description:

Wasn't sure if I should be reporting these errors or if they are captured automatically.

**Comment by tidusjar** [comment]
Did you unblock the .zip file before you extracted?

**Comment by Nurvx** [comment]
Hi @tidusjar!  I didn't unblock the .zip at first (forgot about that) which just caused Ombi.exe to crash right away.  Then I remembered to unblock THEN extract.  I deleted the bad extract folder, then extracted the unblocked .zip into a new directory.  Got Ombi.exe to run and was able to configure everything again.  As I was configuring, I browsed the logs and saw those errors.  Like I mentioned, everything seems to be working, just didn't know what the impacts would be to continue with those errors.
